server/polar/event/endpoints.py

```diff
@@ -138,6 +146,100 @@ async def list(
     )
 
 
+@router.get(
+    "/hierarchy-stats",
+    summary="Get Hierarchy Statistics",
+    openapi_extra={"parameters": [get_metadata_query_openapi_schema()]},
+)
+async def get_hierarchy_stats_endpoint(
```

Comment:
@emilwidlund This endpoint might have been what we were discussing the other day